### PR TITLE
fix: wallet connect and account pages

### PR DESCRIPTION
**Pull request by 2075**
- resolves https://github.com/gamedaoco/gamedao-haiku/issues/368
- cleans up various design glitches and inconsistencies
- sync all pfp, verified icons
- add discord, web3name

Follow up in future tickets:
- account should really get a complete refactor, will create new issue for that
- add translation keys
- add w3n flow
- add discord flow

### CODE REVIEW

File: pkg/app/src/theme/overrides/Button.ts [hunk 1]
```diff
@@ -11,70 +11,81 @@ declare module '@mui/material/Button' {
...
-						color: 'primary' ? theme.palette.common.white : theme.palette.text.secondary,
-					},
-				},
-				{
-					props: { variant: 'secondary' },
-					style: {
-						color: theme.palette.common.white,
-					},
-				},
+				// {
```
Comment (2075) [inline pkg/app/src/theme/overrides/Button.ts:16]: not sure yet if these are still needed

**Comment by 2075** [comment]
@DarkNebula0 @PAkhidue please have a look. cannot seem to fix the typegen from here.

**Comment by 2075** [comment]
@DarkNebula0 @PAkhidue please have a look. cannot seem to fix the typegen from here.

**Comment by PAkhidue** [comment]
> @DarkNebula0 @PAkhidue please have a look. cannot seem to fix the typegen from here.

what typegen? Do you have any error or smth. that helps us to understand the problem? Do you simply run the command "yarn build:graphql"?

If yes, it's because the "web3name" and "discord" field is not present in the GraphQL. We should create a ticket to add these values to our indexer but also Hasura to expose them.

**Comment by 2075** [comment]
> 



> 

yes, that is obviously the case, where to add addtional values aside the packages in the repo? pls link @PAkhidue

**Comment by PAkhidue** [comment]
> > 
> 
> > 
> 
> yes, that is obviously the case, where to add addtional values aside the packages in the repo? pls link @PAkhidue

It's also in this package. You need to add a new migration to the squid-package and add these two fields to the "identity" table. Also you need to update the identity handler to handle the two new additional fields.
Here you go: https://github.com/gamedaoco/gamedao-haiku/blob/main/pkg/squid/src/handlers/identity/events/identitySetHandler.ts

It also looks like there is a field called "web" instead of "web3name". So that means we only need to add the "discord" field.

**Comment by 2075** [comment]
@PAkhidue is this what you would do or what you see as a diff based on the code? I actually added all these fields already, plus web and web3name are two different things :) web is for www and web3name for kilts web3name :)

**Comment by 2075** [comment]
@DarkNebula0 @PAkhidue please lets fix this together, merging other stuff to main will just increase the diff.

**Review by DarkNebula0 — APPROVED** [review]

**Pull request merged**